mocks/mockData.js

```diff
@@ -388,6 +388,21 @@ const nightHoursAgreementTarget = {
   ],
 };
 
+const weekendHoursAgreementTarget = {
```

Comment:
Now we have a dynamic test for all the page titles can you remove the specific ones for weekend and night hours please? The target hours will still be needed.

Reply:
And the associated tests